source/01_release_notes/0100.rst

```diff
@@ -0,0 +1,103 @@
+.. _h-what-is-reference-label:
+
+Release Notes EHRbase 0.10.0 (alpha)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the alpha release of EHRbase (2020-01-10). This means that the basic set of functions has been implemented. Please be aware that the 
+current state can be used to develop openEHR applications but should not be used for real patient data in production. 
+
+The following changes are included in this version:
+
+- openEHR REST API DIRECTORY Endpoints
+- openEHR REST API EHR_STATUS Endpoints (including other_details)
+- Spring Transactions: EHRbase now ensures complete rollback if part of a transaction fails.
+- Improved Template storage: openEHR Templates are stored inside the postgres database instead of the file system (including handling of duplicates)
+- AQL queries with partial paths return data in canonical json format (including full compositions)
+- Multimedia data can be correctly stored and retrieved
+- Spring configuration allows setting the System ID
+- Validation of openEHR Terminology (openEHR terminology codes are tested against an internal terminology service)
+- Order of columns in AQL result sets are now reliably preserved (https://github.com/ehrbase/ehrbase/issues/37)
```

Comment:
Do we want to start thanking people for contributions? I know this practice from, for instance,
- https://github.com/swaywm/sway/releases
- https://github.com/springfox/springfox/releases

I think it would be important to do so one way or another. But what way (in release notes, general contributors list, ...) is another discussion.